Makefile

```diff
@@ -30,10 +30,14 @@ format_net:
 		$(MAKE) -C $(PROJECT) format_net && \
 	) true
 
-format_java:
-	$(foreach PROJECT, $(PROJECTS), \
-		$(MAKE) -C $(PROJECT) format_java && \
-	) true
+format_java_misc:
+	npx prettier --plugin=prettier-plugin-java . --write
+
+format_java_misc-check:
+	npx prettier --plugin=prettier-plugin-java . --check
+
+setup_prettier:
+	npm i --no-save prettier prettier-plugin-java
```

Comment:
Nit: You may not appreciate this,
but not everyone has `npx` or `npm` on their path.

I think this fine. 
Polymorph requires a collection of development tools to operate
(make, gradle, dotnet, dafny, duvet, and more?).

We are growing that list here.
At some point, we should document that list.

Reply:
I agree. The ecosystem is getting large.
I think we should document and embrace it rather than fix it.
Because with so many languages ... :)
cc @robin-aws